website/src/css/customTheme.scss

```diff
@@ -878,8 +878,7 @@ aside[class^="theme-doc-sidebar-container"] {
   }
 
   .menu__link--sublist {
-    font-size: 15px;
-    font-weight: 600;
+    font-size: 17px;
```

Comment:
Same story here, this alters the `font-size` of certain sidebar elements, not only for mobile unfortunately.

To ensure that you only target the elements that you want, you probably need to put this change inside correct `@media` query.